README.md

```diff
@@ -171,6 +171,18 @@ following:
 provides a read-only GraphQL interface to the Registry API. It can be run with
 a local or remote `registry-server`.
 
+### Optional: Connecting to a PostgreSQL database on CloudSQL
+
+[config/cloudsql-postgres.yaml](config/cloudsql-postgres.yaml) contains the
+configuration to connect to a PostgreSQL database hosted on CloudSQL. If you
+don't have an existing PostgreSQL instance, you can follow the
+[instruction](https://cloud.google.com/sql/docs/postgres/quickstart) to setup
```

Comment:
Can we say "these instructions" instead of "the instruction"?

Also, please run ./tools/pretty.sh from the root of the repo to automatically format Markdown files to fit with 80 columns (you'll need to have https://prettier.io/ installed).

Reply:
Done. I didn't notice we have formatter available in the repo